pyproject.toml

```diff
@@ -129,7 +129,7 @@ packages = ["/cosmos"]
 dependencies = [
     "astronomer-cosmos[tests]",
     "apache-airflow-providers-cncf-kubernetes>=5.1.1",
-    "apache-airflow-providers-amazon>=3.0.0,<8.20.0", # https://github.com/apache/airflow/issues/39103
+    "apache-airflow-providers-amazon[s3fs]>=3.0.0,<8.20.0", # https://github.com/apache/airflow/issues/39103
```

Comment:
Do we still need this upper bound limit? It seems the issue was merged in May

Reply:
I have added extras for the cloud providers. Still need to check if we can remove the upper bound and also recap on why exactly it was added.